Big-Spender/readme.md

```diff
@@ -68,39 +71,51 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT * FROM spends
+WHERE LOWER(description) LIKE '%fee%';
```

Comment:
yep, lower works here. Also can consider using` ilike` which would be case insensitive